modules/wmco-prerequisites.adoc

```diff
@@ -7,6 +7,35 @@
 
 The following information details the supported platform versions, Windows Server versions, and networking configurations for the Windows Machine Config Operator. See the vSphere documentation for any information that is relevant to only that platform.
 
+[id="wmco-prerequisites-supported-8.1.1_{context}"]
+== WMCO 8.1.1 supported platforms and Windows Server versions
+
+The following table lists the link:https://docs.microsoft.com/en-us/windows/release-health/windows-server-release-info[Windows Server versions] that are supported by WMCO 8.1.1, based on the applicable platform. Windows Server versions not listed are not supported and attempting to use them will cause errors. To prevent these errors, use only an appropriate version for your platform.
```

Comment:
```suggestion
The following table lists the link:https://docs.microsoft.com/en-us/windows/release-health/windows-server-release-info[Windows Server versions] that are supported by WMCO 8.1.1, based on the applicable platform. Windows Server versions not listed are not supported, and attempting to use them will cause errors. To prevent these errors, use only an appropriate version for your platform.
```